Application/Features/BookingFeature/Commands/CreateBookingCommand.cs

```diff
@@ -0,0 +1,52 @@
+﻿using Application.Interfaces;
+using AutoMapper;
+using Domain.Models;
+using FluentValidation;
+using MediatR;
+
+namespace Application.Features.BookingFeature.Commands
+{
+    public class CreateBookingCommandRequest : IRequest<CreateBookingCommandResponse>
+    {
+        public int UserId { get; set; }
+        public DateTimeOffset StartDate { get; set; }
+
+        public DateTimeOffset EndDate { get; set; }
+        public bool IsRecurring { get; set; }
+        public int Frequency { get; set; }
+        public int WorkplaceId { get; set; }
+    }
+        
+    public class Validator : AbstractValidator<CreateBookingCommandRequest>
+    {
+        public Validator()
+        {
+            RuleFor(x => x.UserId).NotEmpty().WithMessage("UserId must not be blank");
```

Comment:
this field will never be empty, otherwise it won't pass the controller, as it not nullable. Even this field filled with 0 means that it's not empty

Reply:
empty is 0, so it can be

Reply:
So, should I remove this validator rule ?

Reply:
no, it's Ok